FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/transport/src/mbedtls_freertos.c

```diff
@@ -0,0 +1,541 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
The amazon-freertos repository contains an mbed TLS wrapper, [iot_tls.h](https://github.com/aws/amazon-freertos/blob/master/libraries/freertos_plus/standard/tls/include/iot_tls.h) and [iot_tls.c](https://github.com/aws/amazon-freertos/blob/master/libraries/freertos_plus/standard/tls/src/iot_tls.c), that provides the functionality of Connect, disconnect, send and receive. Instead of creating a new wrapper implementation, have we considered just using the same file?

Reply:
Yea. Looked into that. It is a good point. We can look into converging that once Secure sockets and it's support implementation is in path for LTS release. Currently I would keep it separate.